app.js

```diff
@@ -1,21 +1,25 @@
-document.addEventListener('DOMContentLoaded', getCount);
-let counter1 = 0;
-let counter2 = 0;
-document.querySelector('.my-pic1').addEventListener('click', function(){ 
-    counter1++;
-    localStorage.setItem('counter1',counter1);
-    document.getElementById('btn1').innerHTML= localStorage.getItem('counter1');
-});
+document.querySelector('.my-pic1').addEventListener('click', function(){Counter('btn1');});
+
+document.querySelector('.my-pic2').addEventListener('click', function(){Counter('btn2');});
 
-document.querySelector('.my-pic2').addEventListener('click', function(){
-    counter2++;
-    localStorage.setItem('counter2',counter2);
-    document.getElementById('btn2').innerHTML= localStorage.getItem('counter2');
+document.addEventListener('DOMContentLoaded', function(){
+      getCount();

```

Comment:
A good idea to fetch count on DOM load.
The regular practice is to move onLoad functions to the bottom of the file.